src/include/os_fs.i

```diff
@@ -114,7 +114,8 @@ __wt_fs_remove(WT_SESSION_IMPL *session, const char *name, bool durable)
 
 	file_system = S2C(session)->file_system;
 	wt_session = (WT_SESSION *)session;
-	ret = file_system->fs_remove(file_system, wt_session, path, durable);
+	ret = file_system->fs_remove(
+	    file_system, wt_session, path, durable ? WT_FS_DURABLE : 0);
```

Comment:
That is a good reminder point I forgot.  These are internal functions, we can change their signature at will anyway if needed in the future.  Maybe add that sentence into a function comment to remind us in the future about why we used the boolean instead of flags.